src/legacy/core_plugins/timelion/public/panels/timechart/schema.ts

```diff
@@ -17,42 +17,49 @@
  * under the License.
  */
 
-require('./flot');
+import './flot';
```

Comment:
Looks like now we don't need that import

Reply:
Discussed